src/test/java/org/jetbrains/bsp/bazel/BazelBspServerTestData.java

```diff
@@ -110,6 +115,20 @@
   static final InverseSourcesResult EXPECTED_INVERSE_SOURCES =
       new InverseSourcesResult(ImmutableList.of(ID_2));
 
+  static final ScalaMainClassesParams SCALA_MAIN_CLASSES_PARAMS =
+      new ScalaMainClassesParams(ImmutableList.of(ID_1));
+
+  static final ScalaMainClassesResult EXPECTED_SCALA_MAIN_CLASSES =
```

Comment:
I think that it would be nice to add more targets here - with empty params and opts or without `main_class`